apps/fz_http/assets/js/live_view.js

```diff
@@ -20,8 +20,7 @@ const channelToken = document
   .getAttribute("content")
 const notificationChannel =
   userSocket.channel("notification:session", {
-    token: channelToken,
-    user_agent: window.navigator.userAgent
```

Comment:
Hmm I think removing this line might break the `Active Sessions` table?

<img width="1616" alt="Screenshot 2023-01-09 at 5 14 33 PM" src="https://user-images.githubusercontent.com/167144/211439738-3eada60a-f450-4fb4-b74c-0b95dfbb3c9c.png">

Reply:
@jamilbk it should not, I'm getting the `user_agent` in `UserSocket` instead. Phoenix has built-in `:user_agent` option in `Phoenix.Socket` `:connect_info` so there is no sense to duplicate it by sending it from JavaScript as a parameter.

Anyway, I'll have a test for it :).

Reply:
Awesome :-)